data/classes/fx.json

```diff
@@ -57,7 +66,27 @@
     },
     {
       "name": "<init>",
-      "descriptor": "(III)V"
+      "descriptor": "(III)V",
+      "javadoc": [
+        "Constructs a {@code BlockPos} using the provided X, Y, and Z coordinates."
```

Comment:
use the fully qualified class name as per standards ;)

Reply:
Do we still need to use fully qualified class names, even in `{@code}` tags?

Reply:
In code tags, they are not resolved, so no. However, why use a code tag instead of a link with a FQN?

Reply:
A stylistic choice, since a `{@link}` tag to `BlockPos` would be redundant (as it would link to the same class it is already in). I've also seen javadocs in the JDK use plain text (`Constructs a BlockPos using the [...]`), but my preference is using the code tag for the monospace font.

Reply:
ah alright. seems good to me then

Reply:
Fair nuff